homeassistant/components/airscape/fan.py

```diff
@@ -0,0 +1,173 @@
+"""Platform for Airscape fan integration."""
+import logging
+import airscape
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+
+# Import the device class from the component that you want to support
+from homeassistant.components.fan import (
+    SUPPORT_SET_SPEED,
+    PLATFORM_SCHEMA,
+    FanEntity,
+    DOMAIN,
+)
+from homeassistant.const import CONF_HOST, CONF_NAME, CONF_TIMEOUT, CONF_MINIMUM
+
+DEFAULT_TIMEOUT = 5
+DEFAULT_MINIMUM = 1
+
+_LOGGER = logging.getLogger(__name__)
+
+# Validation of the user's configuration
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+    {
+        vol.Required(CONF_HOST): cv.string,
+        vol.Required(CONF_NAME): cv.string,
+        vol.Optional(CONF_TIMEOUT, default=DEFAULT_TIMEOUT): cv.positive_int,
```

Comment:
Can we choose a timeout that is appropriate instead of adding a config option? We try to avoid options for timeouts.